yt_dlp/extractor/orf.py

```diff
@@ -604,10 +608,33 @@ def _extract_video(self, video_id, display_id):
                 formats.extend(fmts)
                 self._merge_subtitles(subs, target=subtitles)
 
+        for subtitle_type in ['vtt']:  # not working formats 'xml', 'srt', 'sami', 'ttml', 'stl'
+            subtitle_url = traverse_obj(api_json, ('_embedded', 'subtitle', f'{subtitle_type}_url'), {str})
+            if subtitle_url is None:
+                continue
+            self._merge_subtitles({
+                'de': [
+                    {
+                        'url': subtitle_url,
+                        'ext': f'{subtitle_type}',
+                    }
+                ],
+            }, target=subtitles)
+
+        age_classification = traverse_obj(api_json, ('age_classification'), {str})
+        age_limit = None
+        if isinstance(age_classification, str) and len(age_classification) != 0:
+            # age_classification is in the format `<age:int>+`
+            # example: "6+" or "18+"
+            age_limit_str = age_classification[:-1]
+            if age_limit_str.isdigit():
+                age_limit = int(age_limit_str)
+
```

Comment:
I think you can just use `parse_age_limit` from `..utils`
If you can just inline this in to the `traverse_obj` call below.

Reply:
thanks for pointing that out, `parse_age_limit` works like it should be :)